opentreemap/exporter/tasks.py

```diff
@@ -208,7 +208,9 @@ def async_csv_export(job, model, query, display_filters):
         write_csv(limited_qs, csv_file,
                   field_order=field_header_map.keys(),
                   field_header_map=field_header_map)
-        filename = generate_filename(limited_qs).replace('plot', 'tree')
+        filename = \
+            generate_filename(limited_qs, append_datestamp=True) \
```

Comment:
When I look at `generate_filename` in djqscsv

https://github.com/azavea/django-queryset-csv/blob/0.3.3/djqscsv/djqscsv.py#L145-L155

Setting `append_datestamp` to True triggers a call to `_append_datestamp`, which is only adding  `%Y%m%d` to the file name.

https://github.com/azavea/django-queryset-csv/blob/0.3.3/djqscsv/djqscsv.py#L203-L215

I assume that I am following the wrong code path. Where is random-looking suffix you mention in the PR description being generated?